examples/gnrc_tcp_cli/Makefile

```diff
@@ -0,0 +1,69 @@
+# name of your application
+APPLICATION = gnrc_tcp_cli
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+PORT = tap1
+#BOARD ?= samr21-xpro
+#PORT = /dev/ttyACM0
+#SERIAL = "ATML2127031800002115"
+#DEFAULT_CHANNEL ?= 17
+#CFLAGS += -DDEFAULT_CHANNEL=$(DEFAULT_CHANNEL)
+
+# Mark Boards with insufficient memory
+BOARD_INSUFFICIENT_MEMORY := airfy-beacon arduino-duemilanove arduino-mega2560 \
+                             arduino-uno calliope-mini chronos microbit sb-430 \
+                             sb-430h nrf51dongle nrf6310 nucleo-f030 nucleo-f042 \
+                             nucleo-f070 nucleo-f072 nucleo-f334 pca10000 \
+                             pca10005 stm32f0discovery telosb weio wsn430-v1_3b \
+                             wsn430-v1_4 yunjia-nrf51822 z1 msb-430 msb-430h
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# PKTBUFFER SIZE
+CFLAGS += -DGNRC_PKTBUF_SIZE=6000
+
+# Override Default MSS, there is a known BUG in the link-layer
+CFLAGS += -DGNRC_TCP_MSS=1220
+
+# Shorten MSL time to speedup testing
+CFLAGS += -DGNRC_TCP_MSL=2500000
+
+# Target Peer Address
+CFLAGS += -DTARGET_ADDR=\"fe80::5c38:e9ff:fe76:6195\"
```

Comment:
btw. why are you setting all values here in the Makefile and have similar `ifdef`s in your main, too?